### ISSUE TITLE
Another atrac3+ problem in Kurohyo 2

### ISSUE THREAD

**Issue by daniel229** [hrydgard/ppsspp#8370]
Looks like since https://github.com/hrydgard/ppsspp/pull/8011 and never fixed
```
43:56:581 sgx-psp-at3- I[ME]: HW\MediaEngine.cpp:87 FF: Frame data doesn't match channel configuration!
43:56:581 sgx-psp-at3- E[ME]: HLE\sceAtrac.cpp:517 avcodec_decode_audio4: Error decoding audio -1094995529 / bebbb1b7
```

Log rename jpg to 7z
![ppsspplog](https://cloud.githubusercontent.com/assets/3481559/12077304/556b6df2-b218-11e5-81fd-a3eda8e90be7.jpg)

**Comment by unknownbrackets** [comment]
So #8435 improves this, but it sometimes loops incorrectly.  That's concerning, because it makes it sound like it's reading data in incorrectly.

I was hoping a2417b4779bdc0b30511506794910c56d81c9f68 would prevent any misreading of data.

Can you log when it loops?  Find this:

```c++
				atrac->SeekToSample(atrac->loopStartSample_ - atrac->FirstOffsetExtra() - atrac->firstSampleOffset_);
				if (atrac->bufferState_ != ATRAC_STATUS_FOR_SCESAS) {
					if (atrac->loopNum_ > 0)
						atrac->loopNum_--;
				}
```

Make it:

```c++
				if (atrac->bufferState_ != ATRAC_STATUS_FOR_SCESAS) {
					if (atrac->loopNum_ > 0) {
						atrac->loopNum_--;
						NOTICE_LOG(HLE, "Looping from %d -> %d", atrac->currentSample, atrac->loopStartSample_ - atrac->FirstOffsetExtra() - atrac->firstSampleOffset_);
					}
				}
				atrac->SeekToSample(atrac->loopStartSample_ - atrac->FirstOffsetExtra() - atrac->firstSampleOffset_);
```

Does it always show the same numbers?  Does it always loop incorrectly when it logs that, or is the incorrect thing at a different time than when it thinks it's looping?

-[Unknown]

**Comment by daniel229** [comment]
I don't see those log in the game,and can't produce the incorrect loop with https://github.com/hrydgard/ppsspp/pull/8435

**Comment by unknownbrackets** [comment]
Ah, sorry - got it.  So this is fixed by #8435, cool.

-[Unknown]
